src/panels/config/automation/action/types/ha-automation-action-choose.ts

```diff
@@ -38,39 +38,41 @@ export class HaChooseAction extends LitElement implements ActionElement {
             )}
             .path=${mdiDelete}
           ></ha-icon-button>
-          <div class="card-content">
-            <h2>
+          <ha-expansion-panel leftChevron>
+            <h3 slot="header">
               ${this.hass.localize(
                 "ui.panel.config.automation.editor.actions.type.choose.option",
                 "number",
                 idx + 1
               )}:
```

Comment:
```suggestion
              )}
```
`:` can be removed as it's an expansion panel title

Reply:
done